.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
     rev: 22.8.0
     hooks:
     -   id: black
+        args: [--line-length=79]
```

Comment:
Recommend synchronizing or deferring to https://github.com/catalystneuro/jazayeri-lab-to-nwb/blob/watters/pyproject.toml#L2, similar to above

Reply:
Synchronized, thanks